issue-pr-templates/lib/util.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const assert = require('assert');
+const Octokat = require('octokat');
+
+module.exports = Util;
+
+function Util(creds) {
+  this.octo = new Octokat(creds);
+};
+
```

Comment:
Add API docs for these utility functions, so that people can utilize them in the future :)